### PR TITLE
Feature/major demon contact

### PR DESCRIPTION
**Pull request by xwhitetower**

### CODE REVIEW

File: localisation/english/demc_demon_windows_l_english.yml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+ demc_demon.plague.temple.blessingwindow.opt.markgeneral:0 "A general should wear the plague brand!"
+ demc_demon.plague.temple.blessingwindow.reply.markgeneral:0 "Your chosen general has been branded! His flesh will endure any pain or desease!"
+ demc_demon.plague.temple.blessingwindow.opt.markadmiral:0 "An admiral should wear the plague brand!"
+ demc_demon.plague.temple.blessingwindow.reply.markadmiral:0 "Your chosen admiral wears the mark of $PANTHEON_MAJOR_PLAGUE$! His rotten flesh will protect him."
+ demc_demon.plague.temple.blessingwindow.opt.bye:0 "I have everything I wanted."
+
+
+
+ demc_demon.pleasure.temple.mainwindow.title:0 "Pleasure temple"
+ demc_demon.pleasure.temple.mainwindow.desc:0 "Hi! The joys of our lord $PANTHEON_MAJOR_PLEASURE$ awaits you in this chamber."
```
Comment (roigcarlo) [inline localisation/english/demc_demon_windows_l_english.yml]: Our lord? **She** is so disapointed with you....

File: localisation/english/demc_demon_windows_l_english.yml [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+ demc_demon.pleasure.temple.blessingwindow.title:0 "Pleasure temple"
+ demc_demon.pleasure.temple.blessingwindow.desc:0 "What blessing of $PANTHEON_MAJOR_PLEASURE$ will you enjoy today?"
+ demc_demon.pleasure.temple.blessingwindow.opt.makemesexy:0 "I want to be sexy!"
+ demc_demon.pleasure.temple.blessingwindow.tooltip.makemesexy:0 "You look stunning! You are still enjoying this blessing!"
+ demc_demon.pleasure.temple.blessingwindow.reply.makemesexy:0 "Your beauty treatment is over. Your skin looks sooooo fantastic now!"
+ demc_demon.pleasure.temple.blessingwindow.opt.makemeloved:0 "I want to be loved!"
+ demc_demon.pleasure.temple.blessingwindow.reply.makemeloved:0 "Your openess has been really well received by your subjects!"
+ demc_demon.pleasure.temple.blessingwindow.opt.markruler:0 "I want the pleasure brand!"
+ demc_demon.pleasure.temple.blessingwindow.reply.markruler:0 "Here you have! You are now a VIP in the eyes of our master!"
```
Comment (roigcarlo) [inline localisation/english/demc_demon_windows_l_english.yml]: Same, is a she isn't it?

File: localisation/english/demc_demon_windows_l_english.yml [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+ demc_demon.generic.temple.mainwindow.desc.fromblessing:0 "Anything else?"
+ demc_demon.generic.temple.mainwindow.desc.upset:0 "The signs show that the gods are angry!"
+ demc_demon.generic.temple.mainwindow.desc.sinner:0 "Sinner! Repent now!"
+ demc_demon.generic.temple.mainwindow.opt.blessings:0 "I want to receive blessings."
+ demc_demon.generic.temple.mainwindow.opt.upset:0 "A sacrifice will please him."
+ demc_demon.generic.temple.mainwindow.opt.sinner:0 "I will atone my sins!"
+ demc_demon.generic.temple.mainwindow.opt.bye:0 "I don't need anything."
+
+
+ demc_demon.generic.temple.blessingwindow.title:0 "Wisdom temple"
```
Comment (roigcarlo) [inline localisation/english/demc_demon_windows_l_english.yml]: Copy paste here, should be `"Temple of the Dark Gods"`

File: common/scripted_triggers/demc_scripted_triggers.txt [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+}
+
+has_sacrificial_energy = {
+	check_variable = {
+		which = sacrifical_energy_count
+		value > 0
+	}
+}
+
+has_2_sacrificial_energy = {
```
Comment (roigcarlo) [inline common/scripted_triggers/demc_scripted_triggers.txt]: What if we need more cases (up to 12)? Is this going to be the final cost of the blessings etc?

File: localisation/english/demc_demon_windows_l_english.yml (hunk 1 above)
Comment (xwhitetower) [reply localisation/english/demc_demon_windows_l_english.yml]: He/She is hermaphrodite or else he/she would not enjoy ALL the pleasures of the flesh

File: common/scripted_triggers/demc_scripted_triggers.txt (hunk 4 above)
Comment (xwhitetower) [reply common/scripted_triggers/demc_scripted_triggers.txt]: Not relevant in my PR, I will not write code I do not need

File: localisation/english/demc_demon_windows_l_english.yml (hunk 3 above)
Comment (xwhitetower) [reply localisation/english/demc_demon_windows_l_english.yml]: fixed

**Pull request merged**